pdfminer/pdffont.py

```diff
@@ -654,6 +655,17 @@ def __init__(self, rsrcmgr, spec, strict=settings.STRICT):
             if strict:
                 raise PDFFontError('Encoding is unspecified')
             name = 'unknown'
+        if type(name) is PDFStream:
```

Comment:
Adding to what @pietermarsman said, we could have a dictionary that contains the mapping. For example - 
`encoding_mapping = {"DLIdent-H": "Identity-H",  "DLIdent-V": "Identity-V"}`. This way we could just add a new mapping without having to write another if-else condition.